src/BloomBrowserUI/bookEdit/toolbox/talkingBook/talkingBook.ts

```diff
@@ -175,7 +175,7 @@ export default class TalkingBookTool implements ITool {
         if (!page) {
             return;
         }
-        if (!this.isImageDescriptionToolActive()) {
+        if (!AudioRecorder.theOneAudioRecorder.showingImageDescriptions) {
```

Comment:
This at least brings the image descriptions display into sync with the slider which is supposed to control them, and stops them from spontaneously turning themselves on and off